packages/react/src/components/Authenticator/SignIn/SignIn.tsx

```diff
@@ -71,7 +83,8 @@ export function SignIn() {
           {isPending ? <>Signing in&hellip;</> : <>Sign In</>}
         </Button>
       </Footer>
-      <Box data-amplify-error>{state.event.data?.message}</Box>
+      <Box data-amplify-error>{actorState.context.remoteError}</Box>
+      <ErrorText amplifyNamespace={amplifyNamespace} />
```

Comment:
We don't need both of these. The `ErrorText` component will look in the `state.context` for the `remoteError`, so if we keep both the `Box` and `ErrorText` here, we will be showing the same error twice.

I think instead, you'll just need to edit the `ErrorText` component to use the new `actorState` method.

Reply:
Right, that's my bad. I did already edit `ErrorText` component, so I'll just remove `Box`.
```suggestion
      <ErrorText amplifyNamespace={amplifyNamespace} />
```